### DISCUSSION TITLE
Mac M1 Support

### DISCUSSION THREAD

**Discussion by dbogatic** [dcajasn/Riskfolio-Lib#31] [Q&A] [answered]
Hello everyone,

I have installed Riskfolio-Lib on Macbook Pro Intel i7 and everything runs smooth. However, when I tried installing on M1 Macbook Pro I keep getting install errors via pip install. I use Miniconda there. Is there a support for M1 at this time? Thank you!

Dragan

**Comment by dcajasn** [comment, marked as answer]
Hi @dbogatic,

I haven't tried Riskfolio-Lib on M1 processor. I build Riskfolio Lib on Amd Bulldozer (2012) and Intel i5 (2012) processors. I read that some python libraries that needs to compile c libraries like cvxpy have issues on M1 computers. I hope that Apple fixes that problem soon.

Best,
Dany
